tests/e2e/core-tests/specs/shopper/front-end-order-email-receiving.test.js

```diff
@@ -29,7 +29,6 @@ const {
 const runOrderEmailReceivingTest = () => {
 	describe('Shopper Order Email Receiving', () => {
 		beforeAll(async () => {
-			await merchant.login();
```

Comment:
We may want to consider keeping this one here. The `await deleteAllEmailLogs();` method below does not log the merchant in:

```
const deleteAllEmailLogs = async () => {
	await merchant.openEmailLog();

	// Make sure we have emails to delete. If we don't, this selector will return null.
	if ( await page.$( '#bulk-action-selector-top' ) !== null ) {
		await setCheckbox( '#cb-select-all-1' );
		await expect( page ).toSelect( '#bulk-action-selector-top', 'Delete' );
		await Promise.all( [
			page.click( '#doaction' ),
			page.waitForNavigation( { waitUntil: 'networkidle0' } ),
		] );
	}
};
```

Reply:
Thanks! I re-added the `merchant.login()` call.

FYI: The reason why the test was passing even without the merchant login is that the jest setup config doesn't seem to log the merchant out, as mentioned by @rrennick below